content/en/docs/01/04_backup.md

```diff
@@ -4,3 +4,143 @@ weight: 14
 sectionnumber: 1.4
 ---
 
+In this lab, you will create scheduled backups for the most important cluster components.
+
+
+## Task {{% param sectionnumber %}}.1: Create user workload backups
+
+As a cluster admin, you can create scheduled backups of your users resources, to be able to restore them in case of emergency, like an accidental deletion.
+
+{{% alert title="Note" color="primary" %}}You will be using Velero to create user workload backups.{{% /alert %}}
+
+Setup a daily backup of the resources in namespace `uptime-app-prod` with a lifetime of 10 days by createing the following schedule:
+
+{{< highlight yaml >}}{{< readfile file="content/en/docs/01/resources/schedule_daily-backup-uptime-app-prod.yaml" >}}{{< /highlight >}}
+
+{{% alert title="Note" color="primary" %}} This resource file is also available at https://raw.githubusercontent.com/acend/openshift-4-ops-training/main/content/en/docs/01/resources/schedule_daily-backup-uptime-app-prod.yaml. {{% /alert %}}
+
+```bash
+oc -n training-infra-velero apply -f https://raw.githubusercontent.com/acend/openshift-4-ops-training/main/content/en/docs/01/resources/schedule_daily-backup-uptime-app-prod.yaml
+```
+
+The schedule will create the first backup immediately. Have a look at the status part of the backup:
+
+```bash
+$ oc -n training-infra-velero describe backup | grep -A 9 Status
+Status:
+  Completion Timestamp:  2021-04-14T09:31:11Z
+  Expiration:            2021-04-24T09:30:50Z
+  Format Version:        1.1.0
+  Phase:                 Completed
+  Progress:
+    Items Backed Up:  31
+    Total Items:      31
+  Start Timestamp:    2021-04-14T09:30:50Z
+  Version:            1
+```
+
+
+## Task {{% param sectionnumber %}}.2 Create etcd backup
+
+To be able to restore the cluster in case of a disaster, you will create a scheduled `etcd` backup.
+
+Create a new project to be used for the backup:
+
+```bash
+oc new-project training-infra-etcd-backup
+```
+
+Since `etcd` is running on the masters, we need to make sure, the cronjob pods creating the `etcd` snapshots are running on the master as well, by adding the following annotation:
```

Comment:
```suggestion
Since `etcd` is running on the masters, we need to make sure the cronjob pods creating the `etcd` snapshots are running on the master as well. We can do this by adding the following annotation:
```